sync/src/types/header_map/mod.rs

```diff
@@ -30,8 +31,8 @@ impl Drop for HeaderMap {
 }
 
 const INTERVAL: Duration = Duration::from_millis(500);
-// key + total_difficulty + skip_hash
-const ITEM_BYTES_SIZE: usize = packed::HeaderView::TOTAL_SIZE + 32 * 3;
+// HeaderIndexView size is 152 bytes
+const ITEM_BYTES_SIZE: usize = 152;
```

Comment:
How about calculate `HeaderIndexView`'s size in compile time by this way instead of hardcode `152`:
```rust
    const ITEM_BYTES_SIZE: usize = std::mem::size_of::<HeaderIndexView>();
```

Reply:
updated code, thanks